### PR TITLE
Add travis checks for translations

### PR DESCRIPTION
**Pull request by rhiaro**
Build fails if there are missing translations.

### CODE REVIEW

**Comment by rhiaro** [comment]
[Refactor OCDS-specific code into a new application](https://trello.com/c/6SyOfmT6/49-refactor-ocds-specific-code-into-a-new-application)

**Comment by rhiaro** [comment]
Oops fixed, thanks @Bjwebb

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +43,10 @@ script:
   - "google-chrome --version"
   - "google-chrome &"
   - coverage run --source cove_ocds,cove_project manage.py test
+  # Check that messages used by OCDS CoVE are translated
+  - "python manage.py makemessages -l es"
+  - "[ \"$TRAVIS_PULL_REQUEST\" = \"false\" ] || [ \"`pocount --incomplete cove/locale/es/LC_MESSAGES/django.po`\" = \"\" ]"
```
Comment (Bjwebb) [inline .travis.yml]: This line should be removed as there is no `cove` directory. (And no translations for `cove_project` either).

**Comment by Bjwebb** [comment]
Now the tests fail on this pull request, because there are non translated strings. I think these have been copied from other bits of cove, so should have translations elsewhere that can be copied. Are you happy to do this, or I could take a look?

**Comment by rhiaro** [comment]
:crossed_fingers: I found the translations, maybe this time?

**Comment by Bjwebb** [comment]
Its still failing, and I'm not sure why.

**Comment by rhiaro** [comment]
Me neither, if I run `pocount` locally the result is `""` as expected

**Comment by odscjames** [comment]
Ok, if you run on Travis not on pull request mode the output is  empty https://travis-ci.com/open-contracting/cove-ocds/builds/147721790#L632

If you run on Travis in pull request mode the output is a table of stats: https://travis-ci.com/open-contracting/cove-ocds/builds/147723592#L640

**Comment by odscjames** [comment]
I'm going to move this over to #14, where I have rebased to master

**Pull request closed without merging**